src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -68,12 +73,20 @@ def copy(
         raise click.ClickException(
             "Both source and target path are local. This operation is not supported."
         )
+    if is_put and recursive:
+        raise click.ClickException("Recursive for PUT is not supported.")
```

Comment:
```suggestion
        raise click.ClickException("Recursive for upload is not supported.")
```
Or should we ignore the flag?

Reply:
I would leave exception, because we will support it soon. I hope 😄 . If we ignore it, command behaviour will change when we finally support it.